llama_stack/core/routers/inference.py

```diff
@@ -611,7 +611,7 @@ async def stream_tokens_and_compute_metrics_openai_chat(
                         completion_text += "".join(choice_data["content_parts"])
 
                     # Add metrics to the chunk
-                    if self.telemetry and chunk.usage:
+                    if self.telemetry and hasattr(chunk, "usage") and chunk.usage:
```

Comment:
this looks unrelated

Reply:
does watsonx return usage information? we need to fix/adapt in the adapter, not in the core. putting this in the core obscures the issue.